internal/pkg/aws/s3/s3.go

```diff
@@ -111,14 +112,18 @@ func (s *S3) EmptyBucket(bucket string) error {
 		if len(objectsToDelete) == 0 {
 			return nil
 		}
-		_, err = s.s3Client.DeleteObjects(&s3.DeleteObjectsInput{
+		delResp, err := s.s3Client.DeleteObjects(&s3.DeleteObjectsInput{
 			Bucket: aws.String(bucket),
 			Delete: &s3.Delete{
 				Objects: objectsToDelete,
+				Quiet:   aws.Bool(true), // we don't care about success values
 			},
 		})
-		if err != nil {
+		switch {
+		case err != nil:
 			return fmt.Errorf("delete objects from bucket %s: %w", bucket, err)
+		case len(delResp.Errors) > 0:
+			return fmt.Errorf("%d/%d objects failed to delete. first failure on key %q: %s", len(delResp.Errors), len(objectsToDelete), aws.StringValue(delResp.Errors[0].Key), aws.StringValue(delResp.Errors[0].Message))
```

Comment:
Love the fraction. 
I can't seem to find guidance on capitalization for multiple-sentence error messages. But I think this slight tweak in wording prevents it from reading like the first of multiple failures of that one key.
```suggestion
			return fmt.Errorf("%d/%d objects failed to delete. First failed on key %q: %s", len(delResp.Errors), len(objectsToDelete), aws.StringValue(delResp.Errors[0].Key), aws.StringValue(delResp.Errors[0].Message))
```